front-end/src/util/__tests__/blockToTime.test.ts

```diff
@@ -0,0 +1,15 @@
+// Copyright 2019-2020 @paritytech/polkassembly authors & contributors
+// This software may be modified and distributed under the terms
+// of the Apache-2.0 license. See the LICENSE file for details.
+import blockToTime from '../blockToTime';
+
+test('blocknumber to time format', () => {
+	const time1 = '1d 12h 40m ';
```

Comment:
Note that this test would fail without a trailing space at the end. (Something to fix in the `blockToTime` script?)

Reply:
yup